config/locales/en.yml

```diff
@@ -227,4 +227,9 @@ en:
     login: "Login"
     logging_in: "Logging you in..."
     forgot_password: "Forgot Password?"
-    get_started: "Get Started"
\ No newline at end of file
+    get_started: "Get Started"
+    enter_info: "Please enter your login information"
+    success: "You've been successfully signed in."
+    user_id: "Your user Id:"
+    errors: 
+      password_email: "Please enter a valid email address and password"
```

Comment:
If you end up deleting where you used these messages, please delete the definitions in these locale files so we don't have them sitting around.

Reply:
Sounds good!